client/components/phone-input/country-flag.jsx

```diff
@@ -49,13 +52,12 @@ export default class extends React.Component {
 
 		if ( countryCode ) {
 			if ( ! this.state.error ) {
```

Comment:
Are there any other errors we want to specificlally guard against? Otherwise, we can probably drop the error handling from this component now.